configure.ac

```diff
@@ -3,7 +3,7 @@ AC_PREREQ([2.60])
 define(_CLIENT_VERSION_MAJOR, 4)
 define(_CLIENT_VERSION_MINOR, 7)
 define(_CLIENT_VERSION_REVISION, 0)
-define(_CLIENT_VERSION_BUILD, 25)
+define(_CLIENT_VERSION_BUILD, 52)
```

Comment:
This would be 4.7.0-2 which is wrong. I don't think the version update needs to be merged back at all; if someone builds on `master` then they should get a post-4.7.0-rc1 build.

Reply:
Exactly the point I was going to make.